### PR TITLE
style: added gap between sideNote sections

### PR DESCRIPTION
**Pull request by shayan-deriv**
changes:
- added gap between title, content and action section
- added `titleClassname` prop to enable the developer to override the styles for it
- updated storybook

### CODE REVIEW

**Review by niloofar-deriv — APPROVED** [review]
LGTM 👍

**Review by amina-deriv — APPROVED** [review]

File: src/components/SideNote/index.tsx [hunk 1]
```diff
@@ -35,17 +36,18 @@ export const SideNote = ({
     className,
     actionClick,
     actionClassName,
+    titleClassName,
```
Comment (amina-deriv) [inline src/components/SideNote/index.tsx:39]: shouldnt we add `titleClassName` to the JSDoc

Comment (shayan-deriv) [reply src/components/SideNote/index.tsx:39]: you're right. I'm adding it now

**Review by yaswanth-deriv — APPROVED** [review]
LGTM

File: src/components/SideNote/index.tsx [hunk 2]
```diff
@@ -25,6 +26,7 @@ type SideNoteProps = Omit<ComponentProps<"div">, "className"> & {
  * @param {SideNoteProps.className} className - Add an extra className to the container.
  * @param {SideNoteProps.actionClick} actionClick - An onclick handler for the action button.
  * @param {SideNoteProps.actionClassName} actionClassName - Add an extra className to the action button.
+ *  * @param {SideNoteProps.titleClassName} titleClassName - Add an extra className to the action button.
```
Comment (utkarsha-deriv) [inline src/components/SideNote/index.tsx:29]: Shouldn't it be `Add an extra className to the title.`?

Comment (shayan-deriv) [reply src/components/SideNote/index.tsx]: done

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**